app/src/main/java/de/westnordost/streetcomplete/screens/about/LogsViewModel.kt

```diff
@@ -0,0 +1,12 @@
+package de.westnordost.streetcomplete.screens.about
+
+import androidx.lifecycle.ViewModel
+import de.westnordost.streetcomplete.data.logs.LogMessage
+import de.westnordost.streetcomplete.data.logs.LogsFilters
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+
+abstract class LogsViewModel : ViewModel() {
+    abstract val filters: MutableStateFlow<LogsFilters>
```

Comment:
Any reason in particular to allow the UI modification to the filters instead of having a `setFilters` function?

Reply:
It is a bit hard to explain what I am thinking.

Essentially, by having a `setFilters` function instead, it makes it clear where I/O between the VM and UI is occurring. This can be enhanced with the VM handling logging of function calls.